torch/fx/experimental/symbolic_shapes.py

```diff
@@ -305,13 +305,6 @@ def uninteresting_files() -> Set[str]:
     return {inspect.getfile(m) for m in mods}
 
 
-# We don't bother with the metaclass as all of the dispatching logic happens
```

Comment:
This comment was introduced in https://github.com/pytorch/pytorch/pull/83330 but never removed when SymDispatchMode was deleted.